stores/governance/src/main/java/com/bloxbean/cardano/yaci/store/governance/processor/CommitteeProcessor.java

```diff
@@ -0,0 +1,86 @@
+
+package com.bloxbean.cardano.yaci.store.governance.processor;
+
+import com.bloxbean.cardano.yaci.core.model.Era;
+import com.bloxbean.cardano.yaci.store.common.config.StoreProperties;
+import com.bloxbean.cardano.yaci.store.common.genesis.ConwayGenesis;
+import com.bloxbean.cardano.yaci.store.common.util.StringUtil;
+import com.bloxbean.cardano.yaci.store.events.EpochChangeEvent;
+import com.bloxbean.cardano.yaci.store.events.RollbackEvent;
+import com.bloxbean.cardano.yaci.store.governance.domain.Committee;
+import com.bloxbean.cardano.yaci.store.governance.storage.CommitteeStorage;
+import com.bloxbean.cardano.yaci.store.governance.storage.CommitteeStorageReader;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.context.event.EventListener;
+import org.springframework.stereotype.Component;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.io.File;
+import java.math.BigInteger;
+
+@Component
+@RequiredArgsConstructor
+@Slf4j
+public class CommitteeProcessor {
+    private final StoreProperties storeProperties;
+    private final CommitteeStorage committeeStorage;
+    private final CommitteeStorageReader committeeStorageReader;
+
+    @EventListener
+    @Transactional
+    public void handleEpochChangeEvent(EpochChangeEvent epochChangeEvent) {
+        Era prevEra = epochChangeEvent.getPreviousEra();
+        Era newEra = epochChangeEvent.getEra();
+        long protocolMagic = epochChangeEvent.getEventMetadata().getProtocolMagic();
+        long slot = epochChangeEvent.getEventMetadata().getSlot();
+        int epoch = epochChangeEvent.getEventMetadata().getEpochNumber();
+
+        // store data from genesis file
+        if (newEra.equals(Era.Conway) && prevEra != Era.Conway) {
+            var latestCommitteeOpt = committeeStorageReader.findByMaxEpoch();
```

Comment:
@Sotatek-HuyLe3a Will this also handle when prevEra == null due to restart or custom network starts with Conway era ?